### PR TITLE
Fixed tiny issue in the Execute component example

### PR DESCRIPTION
**Pull request by andriichumak**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- gooddata-ui-documentation-validation : SKIPPED
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/506/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 8s

File: docs/50_custom__create_new_visualization.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ function CustomVisualization() {
                         {slices?.map((slice, idx) => {
                             // for each slice (row), print the header and then the actual formatted data points
                             return (
-                                <div key="idx">
+                                <div key={idx}>
                                     {slice.sliceTitles().join(">")} -{" "}
```
Comment (plavemar) [inline docs/50_custom__create_new_visualization.md:45]: Please fix this in version 8.0.0 and 8.4.0 as well.

Comment (andriichumak) [reply docs/50_custom__create_new_visualization.md:45]: Fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/840/ : SUCCESS in 1m 45s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/553/ : SUCCESS in 11s

**Review by plavemar — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR440

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/844/ : SUCCESS in 1m 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/152/ : SUCCESS in 55s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/50/ : SUCCESS in 48s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/240/ : SUCCESS in 16s

**Pull request merged**